src/app/dashboard/components/dashboard-new-cases/dashboard-new-cases.component.scss

```diff
@@ -1 +1,111 @@
-/* stylelint-disable */
+@use '../../../../variables.scss' as *;
+
+.dashboard-new-cases {
+  h1 {
+    color: map-get($sormas-accent, 500);
+    font-weight: 500;
+    .nr-of-cases {
+      color: $black;
+    }
+  }
+  .cases-container {
+    display: flex;
+    .left-part {
+      flex: 49%;
+      margin-right: 2%;
```

Comment:
this doesn't look precise;
it would be better to use a flex-basis of 50% for the columns (or whatever the ratio is) and specify a gap of 24px, like in the design (you can also use grid for this)

Reply:
Resolved.